react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -97,21 +96,20 @@ const CreateMapModal = ({
           }}
           validationSchema={validationSchema}
           onSubmit={handleSubmit}
+          initialStatus={{ oldName: '' }}
         >
-          {({ values, setFieldValue }) => {
-            useEffect(() => {
```

Comment:
Refactored this logic to remove the `useEffect` hook being used inside this callback, to resolve an eslint react hooks error.